### PR TITLE
add gnmireverse md

### PR DESCRIPTION
**Pull request by UchihaItachiSama**
convert gnmi reverse EOSC article to md #10 

https://github.com/aristanetworks/openmgmt/issues/10

### CODE REVIEW

**Comment by sulrich** [comment]
useful add, thanks!  

could you find a place to link to this from within the balance of the documentation hierarchy?  i think integrating this into the mix will also require updates to the index, etc. ?

File: docs/telemetry/adapters/gnmireverse/index.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+- [Prerequisite](#prerequisite)
+- [Installing and configuring gNMIReverse client](#installing-and-configuring-gnmireverse-client)
+    - [Building the client and server binaries](#building-the-client-and-server-binaries)
+- [gNMIReverse client daemon Configuration](#gnmireverse-client-daemon-configuration)
+    - [Non-default VRF](#non-default-vrf)
+    - [Default VRF](#default-vrf)
+- [Running the gNMIReverse Server](#running-the-gnmireverse-server)
+- [Subscribing to eos_native paths](#subscribing-to-eos_native-paths)
+- [Subscribing to Smash paths](#subscribing-to-smash-paths)
+- [References](#references)
```
Comment (sulrich) [inline docs/telemetry/adapters/gnmireverse/index.md:18]: this can be removed.  the TOC will be automatically generated by mkdocs.

**Review by sulrich — CHANGES_REQUESTED** [review]
remove the TOC and we can update the `mkdocs.yml` and publish!

File: docs/telemetry/adapters/gnmireverse/index.md (hunk 1 above)
Comment (UchihaItachiSama) [reply docs/telemetry/adapters/gnmireverse/index.md]: Removed TOC from `docs/telemetry/adapters/gnmireverse/index.md`

**Comment by UchihaItachiSama** [comment]
Requested changes committed `docs/telemetry/adapters/gnmireverse/index.md`

**Review by sulrich — APPROVED** [review]
great add!

**Pull request merged**